wallet-test/src/test/scala/org/bitcoins/wallet/WalletUnitTest.scala

```diff
@@ -40,6 +49,89 @@ class WalletUnitTest extends BitcoinSWalletTest {
     }
   }
 
+  it should "know what the last address index is" in { walletApi =>
+    val wallet = walletApi.asInstanceOf[Wallet]
+
+    def getMostRecent(
+        chain: HDChainType,
+        acctIndex: Int
+    ): Future[AddressDb] =
```

Comment:
As code style i would prefer to wrap things like this in `{}` and use dot notation on the `.map`